.github/workflows/podman-test.yaml

```diff
@@ -0,0 +1,26 @@
+name: GitHub Actions Demo
+
+on:
+  pull_request:
+    branches:    
+      - main
+
+jobs:
+  Explore-GitHub-Actions:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: '^1.18'
+
+    - name: Build odo
+      run: make install
+
+    - name: Run Integration tests
+      run: |
+        make test-integration-podman
```

Comment:
Because we are also generating JUnit reports, I think it would be nice also to upload/display such reports, as it would make analysis much easier.
So I was thinking of a final step to either [upload](https://github.com/actions/upload-artifact) the JUnit report as an artifact or, better, parse and display it (e.g., via [`actions/test-reporter`](https://github.com/marketplace/actions/test-reporter))? This new step would run regardless of the status of the "Run Integration tests" step.
(But no problem, this can be done later on in a separate issue/PR)